src/odemis/driver/npmc.py

```diff
@@ -710,7 +710,7 @@ def _updateCurrentPosition(self):
             self._pos = cur_position
 
     def _sendAnswer(self, ans):
-        self._output_buf += "%s\r" % (ans,)
+        self._output_buf += ("%s\r" % (ans,)).encode()
```

Comment:
I don't think `.encode()` should be necessary here. It receives bytes (or should), and append them to bytes.

Reply:
I sometimes passed ints and floats to the function, which is why I needed the encode, but, indeed, it's better to only send bytes. I made the changes and created a separate pull request. I will delete the commit from this PR, so the 2 drivers are separate.